packages/components/src/components/post-mainnavigation-toplevel-item/post-mainnavigation-toplevel-item.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, Element, h, Host, State } from '@stencil/core';
+import { version } from '@root/package.json';
+
+@Component({
+  tag: 'post-mainnavigation-toplevel-item',
```

Comment:
We should wait with implementing this PR until we have a resolution on the name.